static/app/views/dashboardsV2/widgetBuilder/buildSteps/visualizationStep.tsx

```diff
@@ -78,6 +80,16 @@ export function VisualizationStep({
 }
 
 const VisualizationWrapper = styled('div')<{displayType: DisplayType}>`
-  overflow: ${p => (p.displayType === DisplayType.TABLE ? 'hidden' : 'visible')};
   padding-right: ${space(2)};
+  ${p =>
+    p.displayType === DisplayType.TABLE &&
+    css`
+      ${TableCell} {
+        /* 24px ActorContainer height + 16px top and bottom padding + 1px border = 38px */
```

Comment:
```suggestion
        /* 24px ActorContainer height + 16px top and bottom padding + 1px border = 41px */
```

Reply:
oh I forgot to update the comment 😁 thanks